src/policy/fees.h

```diff
@@ -190,6 +190,9 @@ class CBlockPolicyEstimator
     /** Empty mempool transactions on shutdown to record failure to confirm for txs still in mempool */
     void FlushUnconfirmed(CTxMemPool& pool);
 
+    /** Calculation of highest target that estimates are tracked for */
+    unsigned int HighestTargetTracked(FeeEstimateHorizon horizon) const;
```

Comment:
Since `nBlocks` and also `CBlockIndex::nHeight` are `int`, why not return `int` here too?